autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -470,12 +469,20 @@ def predict_and_submit(self, X_test: DataFrame, save=True, save_proba=False):
         y_pred_proba = self.predict_proba(X_test=X_test, inverse_transform=False)
         return self.submit_from_preds(X_test=X_test, y_pred_proba=y_pred_proba, save=save, save_proba=save_proba)
 
-    def leaderboard(self, X=None, y=None, silent=False):
+    def leaderboard(self, X=None, y=None, only_breakpoints=False, silent=False):
         if X is not None:
             leaderboard = self.score_debug(X=X, y=y, silent=True)
         else:
             trainer = self.load_trainer()
             leaderboard = trainer.leaderboard()
+        if only_breakpoints:
+            if 'score_test' in leaderboard.columns and 'pred_time_test' in leaderboard.columns:
+                score_col = 'score_test'
+                inference_time_col = 'pred_time_test'
+            else:
+                score_col = 'score_val'
+                inference_time_col = 'pred_time_val'
```

Comment:
shouldn't these be the `_full` times?

Reply:
These were renamed in the PR that introduced them:

pred_time_val_full -> pred_time_val
pred_time_val -> pred_time_val_marginal